samples/js/browser/DialogServiceConnector/connector.js

```diff
@@ -0,0 +1,115 @@
+// subscription key and region for speech services.
+var subscriptionKey, serviceRegion;
+var SpeechSDK;
+var connector;
+var audioPlayer;
+
+function audioReadLoop(audioStream) {
+  const audioBuffer = new ArrayBuffer(320);
```

Comment:
1. To save you some time, you'll need 
`// <disable>JS1001.SyntaxError</disable>`
at the top of this file,
`// </disable>` at the bottom for ModernCop, 
and a new entry in the codetag exception file for DocFx (I'll msg you the link). 

2. Since, with those codetags, we're allowed to use modern JS, remove all the vars and use const/let instead, and use `() => {` for all function parameters below (instead of `function() {` )